module/zfs/dmu_object.c

```diff
@@ -341,4 +384,10 @@ EXPORT_SYMBOL(dmu_object_free);
 EXPORT_SYMBOL(dmu_object_next);
 EXPORT_SYMBOL(dmu_object_zapify);
 EXPORT_SYMBOL(dmu_object_free_zapified);
+
+/* BEGIN CSTYLED */
+module_param(dmu_object_alloc_chunk_shift, int, 0644);
+MODULE_PARM_DESC(dmu_object_alloc_chunk_shift,
+	"CPU-specific allocator grabs 2^N objects at once");
```

Comment:
The `0644` in the `module_param()` call is for the mode bits of the entry in `/sys/module/zfs/parameters/`.  Changing it to `0444` would make it only settable at module load time.  But in general it's nice to have everything easily modifiable at runtime so I'm happy leaving it as is.  I just wanted to double check my understanding this was OK.